docker-compose.yml

```diff
@@ -217,6 +217,8 @@ services:
       genepinet:
         aliases:
           - nextstrain.genepinet.localdev
+    volumes:
+      - ./src/backend:/usr/src/app
```

Comment:
This means we can run a nextstrain container locally with our local code changes in it.